notebooks/MAST/astroqueryMAST/beginner_program_id_search/beginner_program_id_search.ipynb

```diff
@@ -0,0 +1,402 @@
+    "\n",
+    "To begin, you should make sure that you have an authorized [MyST Account](https://archive.stsci.edu/registration/index.html). \n",
+    "\n",
+    "In order to access data programatically we will also need to obtain an API token. To create and view tokens associated with your account, visit https://auth.mast.stsci.edu/tokens.\n",
+    "\n",
+    "There are several ways to enter your token, including:\n",
+    "1. Manual response to prompt from `Observations.login()` (must be done every time)\n",
+    "2. Python keyring; either through the `keyring` library or `Observations.login`\n",
+    "3. Storing it in the bash environment variable `$MAST-API-TOKEN`\n",
+    "\n",
+    "This flexiblility can overwhelming at first; let's take a look at some examples of these methods below."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Option 1: Respond to prompt\n",
+    "Observations.login()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This works well for infrequent API users, but storing the token is far more convenient for repeated logins. You can store the token using [keyring](https://pypi.org/project/keyring/) or use the built-in `store_token` flag:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Option 2: Store Token\n",
+    "Observations.login(store_token=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Using 'store_token' will allow us to automatically log in, without needing to re-enter the token, for as long as the token remains valid. Note that tokens expire after 10 days of inactivity, or 60 days after creation, whichever comes first. Once it expires, you should use `reenter_token=True` to overwrite the old token with the new one.\n",
+    "\n",
+    "The third option is to store the token as the bash environment variable `$MAST_API_TOKEN`. This method varies from system to system; for more details, you can check out [this guide](https://www3.ntu.edu.sg/home/ehchua/programming/howto/Environment_Variables.html) (links to a non-STScI site).\n",
+    "\n",
+    "Let's take a minute to verify that our login was successful:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "session_info = Observations.session_info()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You should see all of your information above. If not, verify that your token and MyST account are active.\n",
```

Comment:
One piece of info here is:

attrib.Jwstcalengdataaccess: false

This is different from being able to query the engineering database in general, which is public now (I think for everything?  Or at least most things?).  This might be a source of confusion for a user if this shows up as False put the Engineering Database is being advertised as "public" for anyone.  Worth a brief note here clarfiying that this value does not mean you are prohibited from querying and downlading public EDB data?